configs/storage/storage.ks.in

```diff
@@ -0,0 +1,87 @@
+text --non-interactive
+lang en_US.UTF-8
+keyboard us
+rootpw  --plaintext 123456
+sshkey --username=root "%SSH_PUB_KEY%"
+firewall --service=ssh
+authselect select minimal
+selinux --enforcing
+timezone --utc UTC
+network --bootproto=dhcp
+poweroff
+
+# storage
+bootloader --location=mbr --driveorder=vda --append="crashkernel=auto consoleblank=0 net.ifnames=0 console=hvc0"
+zerombr
+clearpart --all
+part /boot --size=256
+part / --grow --size=6120 --fsoptions="noatime"
+part swap --size 4096
+part /home --size=128
+part /tmp --size=128
+part /var --size=4096
```

Comment:
but why not use different dir then? I'd rather reconfigure storage to use e.g. /var/tmp or some other dir in / than carrying a separate ks

Reply:
I've spent a couple of hours on trying to use `/var/tmp` - it's not that easy... Did 2 attempts - first one was about passing `--root /var/tmp/podman-root` argument everywhere, but the grid as a whole was still failing for me. The second one was about simply replacing `/var/lib/containers`  with a symlink, but that caused the pod not to work too.

We don't really need for the storage kickstart to follow changes in other kickstarts - the real underlying OS is interchangeable, we don't care about security policies there etc. I don't want to spend too much time fighting with podman.

I can change the patch not to increase the overall base size and just take some of the space from `/var/tmp` and use it in `/var`.